### DISCUSSION TITLE
ZMK Bluetooth Reset

### DISCUSSION THREAD

**Discussion by duncanam** [manna-harbour/miryoku#272] [Q&A] [answered]
I see that Media Layer has 4 locations for bluetooth connection, is there a binding I am missing for BT_CLR?

**Comment by manna-harbour** [comment, marked as answer]
See https://github.com/manna-harbour/miryoku/tree/master/docs/reference#media:

> Bluetooth profile selection BT 0, BT 1, BT 2, and BT 3 are on the index, middle, ring, and pinkie column keys, respectively. Combine with Shift to select and clear the corresponding profile.

This will be mentioned in the ZMK docs in future, see https://github.com/manna-harbour/miryoku/issues/90.

**Reply by duncanam** [reply]
Brilliant, apologies for missing this. I knew it had to exist. :)
